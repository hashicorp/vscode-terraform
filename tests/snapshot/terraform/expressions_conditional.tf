var.a != "" ? var.a : "default-a"
var.example ? tostring(12) : "hello"
var.example ? 12 : "hello"
