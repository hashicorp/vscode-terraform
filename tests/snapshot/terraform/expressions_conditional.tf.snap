>var.a != "" ? var.a : "default-a"
#^^^ source.terraform support.constant.terraform
#   ^ source.terraform keyword.operator.accessor.terraform
#    ^ source.terraform variable.other.member.terraform
#     ^ source.terraform
#      ^^ source.terraform keyword.operator.terraform
#        ^ source.terraform
#         ^ source.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#          ^ source.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
#           ^ source.terraform
#            ^ source.terraform keyword.operator.terraform
#             ^ source.terraform
#              ^^^ source.terraform support.constant.terraform
#                 ^ source.terraform keyword.operator.accessor.terraform
#                  ^ source.terraform variable.other.member.terraform
#                   ^ source.terraform
#                    ^ source.terraform
#                     ^ source.terraform
#                      ^ source.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#                       ^^^^^^^^^ source.terraform string.quoted.double.terraform
#                                ^ source.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>var.example ? tostring(12) : "hello"
#^^^ source.terraform support.constant.terraform
#   ^ source.terraform keyword.operator.accessor.terraform
#    ^^^^^^^ source.terraform variable.other.member.terraform
#           ^ source.terraform
#            ^ source.terraform keyword.operator.terraform
#             ^ source.terraform
#              ^^^^^^^^ source.terraform meta.function-call.terraform support.function.builtin.terraform
#                      ^ source.terraform meta.function-call.terraform punctuation.section.parens.begin.terraform
#                       ^^ source.terraform meta.function-call.terraform constant.numeric.integer.terraform
#                         ^ source.terraform meta.function-call.terraform punctuation.section.parens.end.terraform
#                          ^ source.terraform
#                           ^ source.terraform
#                            ^ source.terraform
#                             ^ source.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#                              ^^^^^ source.terraform string.quoted.double.terraform
#                                   ^ source.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>var.example ? 12 : "hello"
#^^^ source.terraform support.constant.terraform
#   ^ source.terraform keyword.operator.accessor.terraform
#    ^^^^^^^ source.terraform variable.other.member.terraform
#           ^ source.terraform
#            ^ source.terraform keyword.operator.terraform
#             ^ source.terraform
#              ^^ source.terraform constant.numeric.integer.terraform
#                ^ source.terraform
#                 ^ source.terraform
#                  ^ source.terraform
#                   ^ source.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#                    ^^^^^ source.terraform string.quoted.double.terraform
#                         ^ source.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>